lib/features/thread/presentation/thread_controller.dart

```diff
@@ -345,6 +347,51 @@ class ThreadController extends BaseController {
     viewStateWorker.call();
   }
 
+  void _handleErrorGetOrRefreshChangesEmail(Exception? exception) {
+    logError('ThreadController::_handleErrorGetOrRefreshChangesEmail(): $exception');
+
+    if (exception is CannotCalculateChangesMethodResponseException) {
+      _cleanEmailCacheAndGetAllEmailAgainAction();
```

Comment:
```suggestion
      await _cachingManager.cleanEmailCache();
      _getAllEmail();
```

Reply:
We should separate the new function for better readability and handling later.